packages/app/lib/utils/index.js

```diff
@@ -31,6 +31,41 @@ class FirebaseUtilsModule extends FirebaseModule {
     return this.native.isRunningInTestLab;
   }
 
+  get androidPlayServices() {
```

Comment:
This use to be called `playServicesAvailability` - the example above in the quick start also uses `playServicesAvailability`

Reply:
Thanks, Mike. Sorting out types now 👍